terraform/staging/main.tf

```diff
@@ -118,9 +118,9 @@ module "staging_cluster" {
   ]
 
   flux_repo_name   = "spack-infrastructure"
-  flux_repo_owner  = "mvandenburgh"
-  flux_branch      = "staging"
-  flux_target_path = "k8s/"
+  flux_repo_owner  = "spack"
+  flux_branch      = "kustomize"
```

Comment:
Should this be `main`?

Reply:
Yeah sorry that was a temporary thing to test this and I forgot to pop that commit off. Fixed now.